extensions/portuguese-primeira-liga/package.json

```
@@ -0,0 +1,54 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "portuguese-primeira-liga",
+  "title": "Portuguese Primeira Liga",
+  "description": "View the latest Portuguese Primeira Liga standings.",
+  "icon": "icon.png",
+  "author": "danielsequeira",
+  "owner": "raycast",
+  "categories": [
+    "Fun"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "standings",
+      "title": "Standings",
```

```suggestion
      "title": "View Standings",
```
Usually it's `<verb> <noun>` structure